From 57a4b7b77dc46df2bbd91a3a4bba0159a13177bf Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@apache.org>
Date: Thu, 21 Aug 2014 18:59:44 +0000
Subject: [PATCH 226/791] HADOOP-10920. site plugin couldn't parse hadoop-kms
 index.apt.vm. Contributed by Akira Ajisaka.

Conflicts:
	hadoop-common-project/hadoop-common/CHANGES.txt

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-2@1619543 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit b3ac808c13ead6513af5d1bd92b3cf8c9dc67a57)

Conflicts:
	hadoop-common-project/hadoop-common/CHANGES.txt
---
 .../hadoop-kms/src/site/apt/index.apt.vm           |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/hadoop-common-project/hadoop-kms/src/site/apt/index.apt.vm b/hadoop-common-project/hadoop-kms/src/site/apt/index.apt.vm
index bb4dac0..860c075 100644
--- a/hadoop-common-project/hadoop-kms/src/site/apt/index.apt.vm
+++ b/hadoop-common-project/hadoop-kms/src/site/apt/index.apt.vm
@@ -106,14 +106,14 @@ Hadoop Key Management Server (KMS) - Documentation Sets ${project.version}
 
 ** KMS Aggregated Audit logs
 
-Audit logs are aggregated for API accesses to the GET_KEY_VERSION,
-GET_CURRENT_KEY, DECRYPT_EEK, GENERATE_EEK operations.
+  Audit logs are aggregated for API accesses to the GET_KEY_VERSION,
+  GET_CURRENT_KEY, DECRYPT_EEK, GENERATE_EEK operations.
 
-Entries are grouped by the (user,key,operation) combined key for a configurable
-aggregation interval after which the number of accesses to the specified
-end-point by the user for a given key is flushed to the audit log.
+  Entries are grouped by the (user,key,operation) combined key for a
+  configurable aggregation interval after which the number of accesses to the
+  specified end-point by the user for a given key is flushed to the audit log.
 
-The Aggregation interval is configured via the property :
+  The Aggregation interval is configured via the property :
 
 +---+
   <property>
-- 
1.7.9.5

